recipe/meta.yaml

```diff
@@ -40,6 +43,7 @@ requirements:
     - vsts-python-api
     - toolz
     - scrypt
+    - shellcheck
```

Comment:
Support for `shellcheck` was added in PR ( https://github.com/conda-forge/conda-smithy/pull/1122 ) some time ago, but the requirement was not included here. Have added it. Though please let me know if I'm missing something

Reply:
It was removed in https://github.com/conda-forge/conda-smithy-feedstock/pull/188/files because of some issue I don't remember. Maybe shellcheck crashed?

Reply:
Hmm...ok. Thanks for that detail. Unfortunate we don't have more history in that PR. Maybe @beckermr remembers more details?

Reply:
I don’t recall either way but we don’t currently use it in conda forge for sure.

Reply:
Was it related to issue ( https://github.com/conda-forge/conda-smithy/issues/1316 )? AFAICT everything there has been fixed ( https://github.com/conda-forge/conda-smithy/issues/1316#issuecomment-1441090892 )

Reply:
I'm not sure either. How about we postpone this to a new release? This release is big enough as is.

Reply:
Sure feel free to revert

Reply:
Have filed issue ( https://github.com/conda-forge/conda-smithy-feedstock/issues/248 ) to track adding `shellcheck` as a dependency

Reply:
According to anaconda.org shellcheck has not been built for ppc64le. Can that be right? That'd be a real problem.

Reply:
Thought `shellcheck` would only be used by the linter (when enabled), which is Linux `x86_64` IIUC. If true, why would Linux `ppc64le` builds matter?

Reply:
We wouldn’t be able to even use smithy on ppc64le regardless of the linter.

Reply:
We could add `shellcheck` to `run_constrained`

Reply:
Side note: Created PR ( https://github.com/conda-forge/shellcheck-feedstock/pull/13 ) to add `ppc64le` to `shellcheck`. Unclear on why there wasn't a bot PR for this. Documented some history about what happened regarding `shellcheck` builds on other architectures.

Reply:
Getting `shellcheck` to work on `ppc64le` appears to be non-trivial. Went ahead and moved to `run_constrained` and added a comment linking to issue ( https://github.com/conda-forge/conda-smithy-feedstock/issues/248 ). This at least provides a little context to recipe reviewers about what is going on there